### ISSUE TITLE
Instance creation serialised when using an iallocator

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#365]
Originally reported of Google Code with ID 309.

```
When using an iallocator, two instance creation requests will be serialised. This is because the first creation will grab all node locks, compute via iallocator the desired node(s), then release all others. But the target node(s) are still locked during the full instance creation, which means another instance create will be blocked.

This results, when creating instances on an empty cluster, in a fully serialised creation ☹
```
Originally added on 2012-11-15 10:55:06 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed by hansmi in 2.7 with opportunistic locking
```
Originally added on 2013-05-08 08:35:34 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
